lib/src/fwupd_client.dart

```diff
@@ -145,6 +150,13 @@ class FwupdClient {
         name: 'DeviceRemoved');
     _deviceRemovedSubscription = deviceRemoved.listen(
         (signal) => _deviceRemoved(signal.values[0].asStringVariantDict()));
+
+    var deviceRequest = DBusRemoteObjectSignalStream(
+        object: _root,
+        interface: 'org.freedesktop.fwupd',
+        name: 'DeviceRequest');
```

Comment:
will this throw with older fwupd versions where the signal doesn't exist? do we need some exception handling here?

Reply:
I don't think so - this will just request signals of this name, which can be none. (Can be easily tested by renaming to an invalid name).

Reply:
This object doesn't actually do anything until it is subscribed to, then it calls `AddMatch` with the appropriate filter on the D-Bus server. So there's definitely no exception at construction time.

Reply:
great! i quickly tested on 20.04 that even calling `listen()` doesn't throw. i somehow imagined it would correspond to calling a function that doesn't exist. never mind then :)

Reply:
Great, thanks for checking!